### PR TITLE
MNTOR-2069-README

### PR DESCRIPTION
**Pull request by mozrokafor**
# References: 
Jira: [MNTOR-2069](https://mozilla-hub.atlassian.net/browse/MNTOR-2069)

# Description
added readme and debug and update snapshots commands to package.json

### CODE REVIEW

**Review by rhelmer — CHANGES_REQUESTED** [review]
Thanks! Would you mind updating this section and linking to this new doc? https://github.com/mozilla/blurts-server/#testing

File: src/e2e/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# How to Run e2e Test
+
+###
```
Comment (pdehaan) [inline src/e2e/README.md:3]: ✂️

File: src/e2e/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+#### Add some additional values to the .env file
+
+`E2E_TEST_ACCOUNT_EMAIL=`
+
+`E2E_TEST_ACCOUNT_PASSWORD=`
+
+`E2E_TEST_BASE_URL=`
+
+`E2E_TEST_ENV=` if this is not set, stage will be used
```
Comment (pdehaan) [inline src/e2e/README.md:31]: I don't see this key in the current .env-dist file, we might want to add it and possibly default it to something (or leave a comment on expected/valid values):

https://github.com/mozilla/blurts-server/blob/fe9cafba924c924deb1c1146ce08b33d0916f228/.env-dist#L131-L134

I was able to get a rough hint by searching for keys in the repo and finding this, but not sure what the allowed values are:
https://github.com/mozilla/blurts-server/blob/fe9cafba924c924deb1c1146ce08b33d0916f228/.github/workflows/playwright.yml#L78

File: src/e2e/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+cp .env-dist .env
+```
+
+#### Add some additional values to the .env file
+
+`E2E_TEST_ACCOUNT_EMAIL=`
+
+`E2E_TEST_ACCOUNT_PASSWORD=`
+
+`E2E_TEST_BASE_URL=`
```
Comment (pdehaan) [inline src/e2e/README.md:29]: might need a comment on what value we expect here. I assume a FQDN, but not sure about trailing slashes, etc.

I was able to search and find a couple examples:
- https://github.com/mozilla/blurts-server/blob/fe9cafba924c924deb1c1146ce08b33d0916f228/src/e2e/utils/helpers.js#L22
- https://github.com/mozilla/blurts-server/blob/fe9cafba924c924deb1c1146ce08b33d0916f228/playwright.config.js#L57
- https://github.com/mozilla/blurts-server/blob/fe9cafba924c924deb1c1146ce08b33d0916f228/src/e2e/utils/helpers.js#L13-L17

File: src/e2e/README.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+`E2E_TEST_ENV=` if this is not set, stage will be used
+
+#### Verify helpers.js to make sure correct urls are being used
+
+#### e2e runs on localhost:6060 by default, inorder to run against other envs, provide the end in command
```
Comment (pdehaan) [inline src/e2e/README.md:35]:
```suggestion
#### e2e runs on localhost:6060 by default, in order to run against other envs, provide the env in command
```

File: src/e2e/README.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+`Update the E2E_TEST_ENV in .env file`
```
Comment (pdehaan) [inline src/e2e/README.md:37]:
```suggestion
Update the `E2E_TEST_ENV` in the .env file
```

**Comment by mansaj** [comment]
Thanks for the PR, LGTM pending @pdehaan 's comments above

File: src/e2e/README.md (hunk 2 above)
Comment (mozrokafor) [reply src/e2e/README.md:31]: good point, updating the .env-dist to include this

File: src/e2e/README.md (hunk 3 above)
Comment (mozrokafor) [reply src/e2e/README.md:29]: will just remove this, as choosing the E2E_TEST_ENV will update this env var

File: src/e2e/README.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
+# How to Run e2e Test
+
+###
+
+#### Install npm packages error will show if node 18.12.0^ is not installed, `nvm install 18.12.0` should fix this.
+
+```sh
+npm install
+```
```
[the remark this answers is not in the document]
Comment (mozrokafor) [reply src/e2e/README.md:9]: updating

File: src/e2e/README.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+#### Generate .env from .env-dist
+
+```sh
+cp .env-dist .env
+```
+
+#### Add some additional values to the .env file
+
+`E2E_TEST_ENV=` <'local' | 'dev' | 'stage'> stage is default, for prod please use GA
```
Comment (Vinnl) [inline src/e2e/README.md:25]: I think you might want to move the backtick for clearer formatting?

```suggestion
`E2E_TEST_ENV=<'local' | 'dev' | 'stage'>` stage is default, for prod please use GA
```

**Review by Vinnl — APPROVED** [review]

**Review by pdehaan — APPROVED** [review]

**Review by pdehaan — APPROVED** [review]

**Review by rhelmer — APPROVED** [review]

**Pull request merged**